tofu/geom/utils.py

```diff
@@ -680,28 +688,78 @@ def _compute_CamLOS2D_pinhole(P=None, F=0.1, D12=0.1, N12=100,
                         'Ves': ['V0']}
             }
 
-# Each config can be called by various names (for benchmark and
-# retro-compatibility), this table stores the available names for each unique
-# config in _DCONFIG
-_DCONFIG_TABLE = {'ITER': 'ITER-V2',
-                  'JET': 'JET-V0',
-                  'WEST': 'WEST-V4',
-                  'A1': 'WEST-V1',
-                  'A2': 'ITER-V1',
-                  'A3': 'WEST-Sep',
-                  'B1': 'WEST-V2',
-                  'B2': 'WEST-V3',
-                  'B3': 'WEST-V4',
-                  'B4': 'ITER-V2',
-                  'NSTX': 'NSTX-V0'}
+# Each config can be called by various names / shortcuts (for benchmark and
+# retro-compatibility), this table stores, for each shortcut,
+# the associated unique name it refers to
+_DCONFIG_SHORTCUTS = {'ITER': 'ITER-V2',
+                      'JET': 'JET-V0',
+                      'WEST': 'WEST-V4',
+                      'A1': 'WEST-V1',
+                      'A2': 'ITER-V1',
+                      'A3': 'WEST-Sep',
+                      'B1': 'WEST-V2',
+                      'B2': 'WEST-V3',
+                      'B3': 'WEST-V4',
+                      'B4': 'ITER-V2',
+                      'NSTX': 'NSTX-V0'}
+
+
+def _get_listconfig(dconfig=_DCONFIG, dconfig_shortcuts=_DCONFIG_SHORTCUTS,
+                    returnas=str):
+    """ Hidden function generating the config names table as a str or dict """
+    assert returnas in [dict, str]
+    dc = {k0: [k0] + sorted([k1 for k1, v1 in dconfig_shortcuts.items()
+                             if v1 == k0])
+          for k0 in sorted(dconfig.keys())}
+    if returnas is dict:
+        return dc
+    else:
+        l0 = np.max([len(k0) for k0 in dc.keys()] + [len('unique names')])
+        l1 = np.max([len(str(v0)) for v0 in dc.values()] + [len('shortcuts')])
+        msg = ("\n\t" + "unique names".ljust(l0) + "\tshortcuts"
+               + "\n\t" + "-"*l0 + " \t" + "-"*l1
+               + "\n\t- "
+               + "\n\t- ".join(["{}\t{}".format(k0.ljust(l0), v0)
+                                for k0, v0 in dc.items()]))
+        return msg
 
-# Default config
-_DEFCONFIG = 'ITER'
+
+def get_available_config(dconfig=_DCONFIG,
+                         dconfig_shortcuts=_DCONFIG_SHORTCUTS,
+                         verb=True, returnas=False):
```

Comment:
Why is returnas a bool here? 
Below there's a check `if returnas is str`... Something looks weird here.